### PR TITLE
Bug Fix: Navbar Functionality Menu Items

### PR DESCRIPTION
**Pull request by Rex-1031**
[Pull Request #27](https://www.loom.com/share/afb5a83a93d04bad8df6aa408c06fcd6?sharedAppSource=personal_library "Pull Request #27")

### What work was done?  
- Organized current menu items in to correct configuration based on the wireframe.
- Added links that are needed to the navbar.
- Routed the links to their respective components.   

## 3-12-2022. 
- Home link was removed from navbar.  
- `key` params were added to new `Menu.Item` tags

### Why was this work done?  
Menu items on the navbar  in the product need to match the  menu items in the wireframe. 

- [Trello Card](https://trello.com/c/jEhVrW3k/61-bugfix-update-navbar-functionality-and-design)
- [Single Case View Wireframe](https://www.figma.com/file/mzjVkhSLknSSumyVNAJij7/Family-Promise-Design-Guide?node-id=10%3A174)

## Type of Change
- [x] Bug Fix. 

## Change Status
- [x] Complete, tested, ready to review, and merge.
- [ ]  Complete, but not tested.
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback.

## Checklist
- [x] My code follows the style guidelines of this project. 
- [x]  I have performed a self-review of my own code. 
- [ ]  My code has been reviewed by at least one peer
- [x] I have commented my code, particularly in hard to understand areas.
- [x] My changes generate no new warnings.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] New and existing unit tests pass locally with my changes.
- [x] There are no merger conflicts.

### CODE REVIEW

File: src/components/common/NavHeader.js [hunk 1]
```diff
@@ -31,15 +31,15 @@ const NavHeader = () => {
...
         </Menu.Item>
         <Menu.Item key="services">
           <Link to="/services" />
           Services
         </Menu.Item>
-        <Menu.Item>
+        <Menu.Item key="client-intake-form">
           <Link to="client-intake-form" />
           Intake
         </Menu.Item>
```
Comment (chavionjackson) [inline src/components/common/NavHeader.js:45]: Nice! We got some keys

**Review by ShariqAli-Dev — APPROVED** [review]
These helped a lot. The current navbar was out of date and the updated tags help a ton with navigation.

**Comment by wild-kard** [comment]
Nice design rework on the navbar.

Bringing the front end more in line with the design documents we have combed through with the stakeholder will give us something new to show for our meeting tomorrow.

**Review by wild-kard — APPROVED** [review]
Nice design rework on the navbar.

Bringing the front end more in line with the design documents we have combed through with the stakeholder will give us something new to show for our meeting tomorrow.

**Review by chavionjackson — APPROVED** [review]

**Pull request merged**